exchanges/exchange.go

```diff
@@ -68,22 +68,22 @@ const (
 	Contact         InternationalBankTransactionType = "contact"
 )
```

Comment:
perhaps define InternationalBankTransactionType as int, and enumerate in const using iota, and generate a String() conversion function using go:generate stringer -type=InternationalBankTransactionType, see also: https://splice.com/blog/iota-elegant-constants-golang/ , https://godoc.org/golang.org/x/tools/cmd/stringer , https://blog.golang.org/generate

Reply:
Out of scope for this PR. You can open a PR with those changes if you want.